djangocms_googlemap/templates/djangocms_googlemap/default/map.html

```diff
@@ -24,9 +24,38 @@
         {% render_plugin plugin %}
     {% endfor %}
 </div>
+{% comment %}
+prevents a disaligne bug of the images in the controls,
+when the map is redrawn (trigger of the event 'cms-content-refresh'),
+taken from googles examples
+{% endcomment %}
+{% addtoblock "css" %}
+    <style type="text/css">
+        .gm-control-active>img {
+            box-sizing: content-box;
+            left: 50%;
+            pointer-events: none;
+            position: absolute;
+            top: 50%;
+            transform: translate(-50%,-50%);
+        }
+        .gm-control-active>img:first-child {
+            z-index: 100;
+        }
+    </style>
+{% endaddtoblock %}
+
 
-{% addtoblock "js" %}<script src="https://maps.googleapis.com/maps/api/js{% if googlemap_key %}?key={{ googlemap_key }}{% endif %}" async defer></script>{% endaddtoblock %}
-{% addtoblock "js" %}<script src="{% static 'djangocms_googlemap/js/djangocms.googlemap.js' %}" async defer></script>{% endaddtoblock %}
+{% addtoblock "js" %}
```

Comment:
Similar to the CSS, please split them apart so that:

```
{% addtoblock "js" %}<script src="https://maps.googleapis.com/maps/api/js{% if googlemap_key %}?key={{ googlemap_key }}{% endif %}" async defer></script>{% endaddtoblock %}
{% addtoblock "js" %}<script src="{% static 'djangocms_googlemap/js/djangocms.googlemap.js' %}" async defer></script>{% endaddtoblock %}
```

is loaded in live mode and:

```
{% addtoblock "js" %}
<script src="https://maps.googleapis.com/maps/api/js{% if googlemap_key %}?key={{ googlemap_key }}{% endif %}"
    class="cms-execute-js-to-render cms-trigger-event-window-load" async defer>
</script>
<script src="{% static 'djangocms_googlemap/js/djangocms.googlemap.js' %}"
    class="cms-execute-js-to-render cms-trigger-event-window-load" async defer>
</script>
{% endaddtoblock %}
```

is loaded on edit. Might also be simpler to add a with around to store the class and then apply as variable to make it more lean.

Reply:
I added a `with tag` to define the class string and it is only used if in edit mode.
The blocks where already split as [recommended](http://docs.django-cms.org/en/latest/how_to/custom_plugins.html?highlight=addtoblock#handling-media) , just the git diff view takes the lines so far apart, one can't see it properly. 
I think also following the guide line `External files should be on one line, with no spaces or newlines between the addtoblock tag and the HTML tags.`, would lead to a decrease in readability. And since these resources are aren't likely to be used by other plugins, it shouldn't cause any problems.

Reply:
I added a `spaceless` block inside of the `addtoblock` block, this should make it virtually identical to writing all in one line.